### ISSUE TITLE
Character comparison change?

### ISSUE THREAD

**Issue by r-i-c-h** [chriscoyier/coding-fonts#69]
I usually use the string `L|l1iI0Oo` when checking out fonts. 

The idea is there at the bottom of the character's tab, but separating the 1/l/O/0 doesn't provide a great comparison when "the whole point" is to see if you can tell the difference between those characters - ie, if the characters are 'well designed', the differences should be apparent/easy-to-spot. Since it is all done with screen shots, not the easiest\quickest PR, and this is basically a question of the core design of that tab, but perhaps food for thought? (Also, of course, regardless of format, I'd like to see my letter-I, |, and i comparisons along the 1s and ls)

addendum: (If not obvious, my suggested test string is capL, pipe-symbol,  lowerL, 1, lowerI, capI, 0, capO, lowerO)

**Comment by manngo** [comment]
Agreed. I think the comparison should certainly include `I` (capital I) and `|` (pipe).

**Comment by chriscoyier** [comment]
Mark Simon on adding _ underscores to this page as well:
https://github.com/chriscoyier/coding-fonts/issues/87

**Comment by chriscoyier** [comment]
We have this change in place for new fonts:
https://coding-fonts.css-tricks.com/fonts/codelia/?language=charmap

I think we need to:
1) Redo all other screenshots
2) Force-clear the cache somehow. Either go into Cloudinary and delete the ones there that have been fetched already, or implement a file-name-change system and put like e`?v=2` on the end of the image sources or something.
